thorlcr/graph/thgraphslave.cpp

```diff
@@ -1671,6 +1671,9 @@ CJobSlave::CJobSlave(ISlaveWatchdog *_watchdog, IPropertyTree *_workUnitInfo, co
         setRemoteOutputCompressionDefault(remoteCompressedOutput);
 
     actInitWaitTimeMins = getOptInt(THOROPT_ACTINIT_WAITTIME_MINS, DEFAULT_MAX_ACTINITWAITTIME);
+    // Increase init time, if maxLfnBlockTime is large
+    if (queryMaxLfnBlockTimeMins() >= actInitWaitTimeMins)
+        actInitWaitTimeMins = queryMaxLfnBlockTimeMins()+1;
```

Comment:
The default LFN block time is 2hrs, i.e. a query will block on a file lock for that long before giving up with an error.
The activity initialization time need to wait at least as long as the configured LFN block time, otherwise the query will abort with the "Aborted whilst waiting to receive actinit data" error before it times out waiting for the file lock.